tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -172,10 +172,6 @@ public function dataFileAsserts(): iterable
 			yield from $this->gatherAssertTypes(__DIR__ . '/../Rules/Comparison/data/bug-9499.php');
 		}
 
-		if (PHP_VERSION_ID >= 70300 && PHP_VERSION_ID < 70400) {
-			yield from $this->gatherAssertTypes(__DIR__ . '/data/preg_match_shapes_php73.php');
```

Comment:
this test is contained in `preg_match_shapes_php74.php` with proper expectations across all php versions. we no longer need separate expectations for 7.2/7.3